### PR TITLE
Closes #2764 Grey out window when resizing

### LINKED ISSUE #2764: [Enhancement] The window should be grayed out when in resize mode
**[Affected Versions]:**
- Firefox Reality version 9-40381434 (eff107e5)

**[Affected Platforms]:**
- Oculus Go - Android 7.1.2
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2

**[Prerequisites]:**
- The Firefox Reality browser is opened.

**[Steps to reproduce]:**
1. Click the Hamburger menu from the navigation bar.
2. Click the “Resize” button from the menu.
3. Observe the resize mode. 

**[Expected results]:**
- The window should gain a gray overlay so the user knows that the content in the window can no longer be interacted with.

**[Actual results]:**
- The window doesn’t have a gray overlay.

**[Notes]:**
- The overlay could also have some arrows pointed to the circles used to manually resize the window and a message informing the user that they can manually resize the window.

org mozilla vrbrowser-20200210-091518

### PR DESCRIPTION
**Pull request by keianhzo**
Closes #2764 Grey out window when resizing

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java [hunk 1]
```diff
@@ -783,6 +785,9 @@ private void exitResizeMode(ResizeAction aResizeAction) {
             TelemetryWrapper.windowsResizeEvent();
             GleanMetricsService.windowsResizeEvent();
         }
+
+        mAttachedWindow.mWidgetPlacement.tintColor = 0xFFFFFFFF;
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java:789]: I wonder if we should change all these hard code hex colors to constants:
```
final int WhiteTint = 0xFFFFFFFF;
final int GrayTint = 0x555555FF;
```

Comment (keianhzo) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java]: Refactored hex colors to `Windows`

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java [hunk 2]
```diff
@@ -1663,7 +1662,7 @@ public void onContextMenu(GeckoSession session, int screenX, int screenY, Contex
         }
         mContextMenu.show(REQUEST_FOCUS);
 
-        mWidgetPlacement.tintColor = 0x555555FF;
+        mWidgetPlacement.tintColor = Windows.GREY;
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java:1665]: nit: The color is `GRAY`, the little gray men from space are `GREYS` (Mostly a joke, I think the US is they only place that spells it `gray`).

Comment (keianhzo) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java]: Updated

**Pull request merged**